packages/create-request-form/src/utils/invoiceTotals.ts

```diff
@@ -0,0 +1,34 @@
+export const calculateInvoiceTotals = (data: InvoiceItem[]) => {
+  let amountWithoutTax = 0;
+  let totalDiscount = 0;
+  let totalTaxAmount = 0;
+  let totalAmount = 0;
+
+  for (const item of data) {
+    const discountAmount = item.discount && item.unitPrice - item.discount;
+    const taxableAmount = (item.unitPrice - item.discount) * item.quantity;
+    amountWithoutTax += taxableAmount;
+    totalDiscount += discountAmount * item.quantity;
+
+    const itemTaxAmount = taxableAmount * (item.tax.amount / 100);
+    totalTaxAmount += itemTaxAmount;
+    totalAmount += taxableAmount + itemTaxAmount;
+  }
+
+  totalAmount = amountWithoutTax + totalTaxAmount;
+
+  return {
+    amountWithoutTax,
+    totalDiscount,
+    totalTaxAmount,
+    totalAmount,
+  };
+};
+
+export const calculateItemTotal = (item: InvoiceItem): number => {
```

Comment:
Consider using `getInvoiceLineTotal` from `data-format`
https://github.com/RequestNetwork/requestNetwork/blob/master/packages/data-format/src/format/rnf_invoice/utils.ts#L11

Reply:
Decided to use custom utils for now.